.github/CODEOWNERS

```
@@ -0,0 +1 @@
+/docs/ @eslint/website-team
```

making sure everyone can approve

```suggestion
/docs/ @eslint/website-team @eslint/eslint-team
```

---

done